packages/common-all/src/types/configs/commands/lookup.ts

```diff
@@ -27,6 +27,7 @@ export type NoteLookupConfig = {
   confirmVaultOnCreate: boolean;
   leaveTrace: boolean;
   bubbleUpCreateNew: boolean;
+  fuzzThreshold?: number;
```

Comment:
the fuse engine's fuzz threshold affects all lookup derivative commands. Should this config be in `commands.lookups.note`?

Reply:
That makes me think... a lot of the configs for note lookup also affects other commands. Not sure what the most natural way to namespace this would be.

Reply:
Yes, it would affect the other commands (also right now as is currently this affects the schema look up as well).